Adding docker-compose

stlimtat opened the pull request:
Hi,
  I did a simple bunch of work to add Dockerfile-sixpack and docker-compose to simplify the deployment of a docker instance of the application.

  I hope you can add it to your repo.

Commit:
Using a redis config to support the docker-compose capabilities

Commit:
Adding a dockerfile for this flask based application

Commit:
Initial copy of docker-compose.yml that works

Commit:
Adding the getting started section to run on docker-compose

Commit:
Fixing docker-compose to include seeding data from the test data

Commit:
config.yml requires the configuration of cors.  It is no longer sufficient to not specify cors as an auxiliary requirement

Commit:
Adding the gitignore portion specified in flask

zackkitzmiller commented on README.rst:
```diff
@@ -19,6 +19,13 @@ Requirements
 * Redis >= 2.6
 * Python >= 2.7 (3.0 untested, pull requests welcome)
 
+Getting Started WIth Docker
```
Small typo here. Capital I

zackkitzmiller commented on config.yml:
```diff
@@ -1,16 +1,22 @@
 enabled: True
 
 redis_port: 6379
-redis_host: localhost
```
Can you revert the changes to config.yml

stlimtat committed:
Big I to small i

stlimtat committed:
Update config.yml

Reverted some changes.  Noted that the cors_origin cannot not exist in the code.

stlimtat committed:
Update config.yml

Adding the word secret

stlimtat committed:
Update config.yml

Defaulting to have metrics set to false.

stlimtat reviewed (commented):
Fixed requested changes.

Commit:
Fixing the Dockerfile and docker-compose file to only specify the location of the config file

Commit:
Adding a docker-config.yml to simplify the implementation of config.yml changes

stlimtat commented:
Ok.  The fixes I think makes the system more generic are in.

TODO:
a. Config will read a file and does not allow the override of specific variables such as SIXPACK_CONFIG_REDIS_HOST to be done via environment which may be useful for deployments such as kubernetes type configuration.

zackkitzmiller reviewed (approved):
This looks good to me.
